flask_mongoengine/wtf/models.py

```diff
@@ -4,12 +4,12 @@
 class ModelForm(FlaskForm):
     """A WTForms mongoengine model form"""
 
-    def __init__(self, formdata=None, obj=None, **kwargs):
-        self.instance = (kwargs.pop('instance', None) or obj)
+    def __init__(self, formdata=None, **kwargs):
+        self.instance = (kwargs.pop('instance', None) or kwargs.get('obj'))
```

Comment:
Ah, without checking, I assumed that the code would produce the same result as ``{'a':1, 'b':2}['d']`` - my bad.